project2/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Overpass:400,700|Questrial|Righteous&display=swap"
+        rel="stylesheet">
+    <script src="https://kit.fontawesome.com/f7ec3810d9.js" crossorigin="anonymous"></script>
+    <title>MOONVIES</title>
+</head>
+
+<body>
+    <h1>MOONVIES</h1>
+    <nav class="nav">
+        <ul>
+            <li><a href="movienav">MOVIES</a></li>
+            <li><a href="shownav">SHOWS</a></li>
+            <li><a href="reviewnav">REVIEWS</a></li>
+            <li><a href="aboutnav">ABOUT US</a></li>
+            <li><a href="accountnav">ACCOUNT</a></li>
+        </ul>
+    </nav>
+
+
+
+    <section class="cardbox">
+        <img src="images/infin.jpg">
```

Comment:
1. Your image is missing `alt` text, so users relying on screenreaders will just hear the name of the file "Images, images infin dot jpg" -- I recommend adding `alt` text to describe this image.
1. Your image is 1200px wide, but the card size maxes out at 640px, so you should resize this image to be closer to the max width, which will create a smaller download for your users.